### ISSUE TITLE
Refreshing browser results in gebberish

### ISSUE THREAD

**Issue by PorreKaj** [ironmansoftware/universal-dashboard#172]
```
$theme = Get-UDTheme Azure

$DashboardPage = New-UDPage -Name "Dashboard" -Icon Home -Content { 
    New-UDCard -Title "Welcome, $Session:User" -Text "This is your custom dashboard." 
}

$Dashboard = New-UDDashboard -Title "IT Automation Dashboard" -Theme $Theme -Page @(
    $DashboardPage
) 
```
Module version 1.6.1
I have a simple Dashboard (above), If I visit the dashboard it looks good, but if I hit F5 I get this:

![image](https://user-images.githubusercontent.com/20628942/40057915-e0eb99b2-584f-11e8-80a7-fc45b816aba1.png)

**Comment by adamdriscoll** [comment]
After looking at this a bit, I see the issue here. If you name your page "Dashboard" this will happen. If you name it something else, like "My Dashboard" it won't happen.

**Comment by PorreKaj** [comment]
Correct, Thanks!

**Comment by adamdriscoll** [comment]
I'll fix it for 1.6.2.

**Comment by adamdriscoll** [comment]
Resolved in 2.2.0
